metaissue.go

```diff
@@ -36,15 +36,22 @@ type MetaIssueType struct {
 	Description string                `json:"description,omitempty"`
 	IconUrl     string                `json:"iconurl,omitempty"`
 	Name        string                `json:"name,omitempty"`
-	Subtasks    bool                  `json:"subtask,omitempty"`
+	Subtask     bool                  `json:"subtask,omitempty"`
```

Comment:
Why did you change this?

Reply:
I changed Subtasks to Subtask to match the jira data model. In addition, it is a boolean to check if the Issue Type is a subtask...so the 's' makes it confusing as it is not subtasks itself.

Reply:
But that will break this library for everyone else that's ever used it.